README.md

```diff
@@ -182,4 +169,14 @@ Sets the frequency at which data is flushed to the server. Resets the interval t
 events.teardown()
 ```
 
-Stops the interval timer, clears the data queue and prevents any further data from being flushed to the server.
+Stops the interval timer, clears the data queue and prevents any further data from being flushed to the server. Useful for testing
+
+#### `events.from`
+
+```js
+const scoped = events.from(ctx);
+```
+
+Returns a scoped version of the events api.
+
+- `ctx: FusionContext` - Required.
```

Comment:
Should link to docs about FusionContext

Reply:
How are we planning on doing those links when we have to handle both open source and closed source documentation?